java/ql/automodel/publish.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+set -e
+
+AUTOMODEL_ROOT="$(dirname $0)"
+WORKSPACE_ROOT="$AUTOMODEL_ROOT/../../../.."
+GRPS="automodel,-test"
+
+if [ -z "$CODEQL_DIST" ]; then
+  echo "CODEQL_DIST not set"
+  exit -1
+fi
+
+cd "$AUTOMODEL_ROOT"
+echo Testing automodel queries
+"${CODEQL_DIST}/codeql" test run test
+
+cd "$WORKSPACE_ROOT"
+
+echo Preparing release
+"${CODEQL_DIST}/codeql" pack release --groups $GRPS
+
+echo Publishing automodel
+"${CODEQL_DIST}/codeql" pack publish --groups $GRPS
+
+echo Bumping versions
+"${CODEQL_DIST}/codeql" pack post-release --groups $GRPS
+
+echo Automodel packs successfully published. Please commit and push the version changes.
```

Comment:
If you wanted to be extra fancy, you could commit and push directly from this script.

Reply:
Hmm... I kind of like it not committing for me, then there is less that suddenly happens. But we can also tweak this afterwards. It almost seems like there should be a generic publish script we could have for all the separate packs, but I am not sure how frequent these separate packs are?